src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6055,16 +6067,16 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_DuplicateRecordConstructor" xml:space="preserve">
     <value>There cannot be a primary constructor and a member constructor with the same parameter types.</value>
   </data>
-  <data name="WRN_GeneratorFailedDuringGeneration" xml:space="preserve">
-    <value>Generator '{0}' failed to generate source. It will not contribute to the output and compilation errors may occur as a result.</value>
+  <data name="ERR_InvalidWithReceiverType" xml:space="preserve">
+    <value>The receiver of a `with` expression must have a valid non-void type.</value>
```

Comment:
>valid [](start = 59, length = 5)

What does "valid" mean here?  I suggest dropping the word.  If it were an error type we presumably would not report this.